sparql/pom.xml

```diff
@@ -33,7 +33,7 @@
 
     <properties>
         <interpreter.name>sparql</interpreter.name>
-        <jena.version>3.12.0</jena.version>
+        <jena.version>3.17.0</jena.version>
```

Comment:
The Java 11 is [required since Jena 4](https://jena.apache.org/download/index.cgi#:~:text=Jena4%20requires%20Java%2011.).